lxd/instance_put.go

```diff
@@ -188,11 +198,21 @@ func instancePut(d *Daemon, r *http.Request) response.Response {
 
 		opType = operationtype.InstanceUpdate
 	} else {
+		disks, err := instance.DisksFromString(configRaw.Disks)
+		if err != nil {
+			return response.BadRequest(err)
+		}
+
+		// "excluded_disks" do not apply on single volume snapshots.
+		if disks == instance.DisksRoot && len(configRaw.ExcludeDisks) > 0 {
+			return response.BadRequest(errors.New(`Field "excluded_disks" was provided on a single volume snapshot`))
```

Comment:
"root volume only snapshots" would be clearer in my view as it aligns with the default "root" Disks Mode